hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/ClusterWideConfigurationService.java

```diff
@@ -133,6 +136,9 @@
             lockConfigs,
             listConfigs,
             setConfigs,
+            atomicLongConfigs,
```

Comment:
Why didn't we see any failures with the missing `atomicLongConfigs` and `atomicReferenceConfigs` before?

The field seems to be used in the context of split-brain healing, which collects all existing dynamic configs and resets the service. Maybe we need a `DynamicConfigSplitBrainTest` which checks that all kind of configurations are correctly merged.

Reply:
That's a good question to @vbekiaris but it's out of scope of this PR, so plz address any changes related to it in another PR.